tests/frontier/vm/vm_test_helpers.py

```diff
@@ -0,0 +1,123 @@
+import json
+import os
+from typing import Any
+
+from ethereum.base_types import U256, Uint
+from ethereum.crypto import keccak256
+from ethereum.frontier import rlp
+from ethereum.frontier.eth_types import Account, State
+from ethereum.frontier.vm import Environment
+from ethereum.frontier.vm.interpreter import process_call
+from tests.frontier.helpers import (
+    hex2address,
+    hex2bytes,
+    hex2bytes32,
+    hex2u256,
+    hex2uint,
+)
+
+
+def run_test(test_dir: str, test_file: str) -> None:
+    test_data = load_test(test_dir, test_file)
+    target = test_data["target"]
+    env = test_data["env"]
+
+    gas_left, logs = process_call(
+        caller=test_data["caller"],
+        target=target,
+        data=test_data["data"],
+        value=test_data["value"],
+        gas=test_data["gas"],
+        depth=test_data["depth"],
+        env=env,
+    )
+
+    assert gas_left == test_data["expected_gas_left"]
+    assert keccak256(rlp.encode(logs)) == test_data["expected_logs_hash"]
+    # We are checking only the storage here and not the whole state, as the
+    # balances in the testcases don't change even though some value is
+    # transferred along with code invokation. But our evm execution transfers
```

Comment:
`s/invokation/invocation`

Reply:
Weird why flake8 didn't catch this. Anyhow fixed this, thanks.